Firestore/core/src/firebase/firestore/model/database_id.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_DATABASE_ID_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_DATABASE_ID_H_
+
+#include <string>
+
+namespace firebase {
+namespace firestore {
+namespace model {
+
+/** A DatabaseId represents a particular database in the Firestore. */
+class DatabaseId {
+ public:
+  /** The default name for "unset" database ID in resource names. */
+  static const char* kDefaultDatabaseId;
```

Comment:
This can be `static constexpr const char* kDefaultDatabaseId = "(default)";`.

You'll still need the definition in the .cc file, but there you need only declare it exists (without the value).

Reply:
done